src/test/java/unstable/ContextMenuUnstableTests.java

```diff
@@ -0,0 +1,103 @@
+package unstable;
+
+import guitests.UITest;
+import javafx.application.Platform;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.GuiTest;
+import ui.components.FilterTextField;
+import ui.components.IssueListView;
+import ui.listpanel.ListPanel;
+import ui.listpanel.ListPanelCell;
+import java.lang.reflect.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContextMenuUnstableTests extends UITest {
```

Comment:
I think this can have a more specific name. It's testing read/unread for the issue list view, so you might want those to feature in there somewhere.